### PR TITLE
Docs: some work on the network map docs.

### PR DESCRIPTION
**Pull request by mikehearn**

### CODE REVIEW

File: docs/source/network-map.rst [hunk 1]
```diff
@@ -31,6 +42,12 @@ The set of REST end-points for the network map service are as follows.
 | GET            | /network-map/network-parameters/{hash}  | Retrieve the signed network parameters (see below). The entire object is signed with the network map certificate which is also attached.     |
 +----------------+-----------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------+
 
+We use HTTP is used for the network map service instead of Corda's own AMQP based peer to peer messaging protocol to
```
Comment (shamsasari) [inline docs/source/network-map.rst]: Remove "We use"?

File: docs/source/network-map.rst [hunk 2]
```diff
@@ -103,12 +142,22 @@ node operator about the event.
...
+The node administrator can review the change and decide if they are going to accept it. The approval should be do
+before the update Deadline. Nodes that don't approve before the deadline will likely be removed from the network map by
+the zone operator, but that is a decision that is left to the operator's discretion. For example the operator might also
+choose to change the deadline instead.
+
+If the network operator starts advertising a different set of new parameters then that new set overrides the previous set.
+Only the latest update can be accepted.
+
+To send back parameters approval to the zone operator, the RPC method ``fun acceptNewNetworkParameters(parametersHash: SecureHash)``
+has to be called with ``parametersHash`` from update. Note that approval cannot be undone. You can do this via the Corda
```
Comment (shamsasari) [inline docs/source/network-map.rst]: from _the_ update

File: docs/source/network-map.rst [hunk 3]
```diff
@@ -85,16 +117,23 @@ Network parameters update process
 ---------------------------------
 
 In case of the need to change network parameters Corda zone operator will start the update process. There are many reasons
-that may lead to this decision: we discovered that some new fields have to be added to enable smooth network interoperability or change
-of the existing compatibility constants is required due to upgrade or security reasons.
+that may lead to this decision: new fields that were added to enable smooth network interoperability, or a change
```
Comment (wawrzek) [inline docs/source/network-map.rst]: Isn't adding/removing Notary the most likely reason to adjust network parameters?

File: docs/source/network-map.rst [hunk 4]
```diff
@@ -1,12 +1,23 @@
...
+to agree on to remain in sync.
+
+.. note:: In Corda 3 no implementation of the HTTP network map server is provided. This is because the details of how
+   a compatibility zone manages its membership (the databases, ticketing workflows, HSM hardware etc) is expected to vary
+   between operators, so we provide a simple REST based protocol for uploading/downloading NodeInfos and managing
+   network parameters. A future version of Corda may provide a simple "stub" implementation for running test zones.
+   In Corda 3 the right way to run a test network is through distribution of the relevant files via your own mechanisms.
+   We provide a tool to automate the bulk of this task (see below).
+
 HTTP network map service
```
Comment (wawrzek) [inline docs/source/network-map.rst]: I would make clear that this paragraph is not part of Corda 3 (I know that's there is a note above):

  HTTP network map service  ->   HTTP network map service (in future releases)

Comment (mikehearn) [reply docs/source/network-map.rst]: I changed the heading to protocol, to make it clear that this is a protocol description and not a description of software.

File: docs/source/network-map.rst [hunk 5]
```diff
@@ -103,12 +142,22 @@ node operator about the event.
...
+If the network operator starts advertising a different set of new parameters then that new set overrides the previous set.
+Only the latest update can be accepted.
+
+To send back parameters approval to the zone operator, the RPC method ``fun acceptNewNetworkParameters(parametersHash: SecureHash)``
+has to be called with ``parametersHash`` from the update. Note that approval cannot be undone. You can do this via the Corda
+shell (see :doc:`shell`):
+
+``run acceptNewNetworkParameters parametersHash: "ba19fc1b9e9c1c7cbea712efda5f78b53ae4e5d123c89d02c9da44ec50e9c17d"``
+
+If the administrator does not accept the update then next time the node polls network map after the deadline, the
```
Comment (kasiastreich) [inline docs/source/network-map.rst]: Restart will happen if we accept the parameters too, node will switch to new parameters set automatically after getting up again.

**Pull request merged**